contracts/src/factory/HyperdriveFactory.sol

```diff
@@ -21,7 +21,9 @@ contract HyperdriveFactory is IHyperdriveFactory {
     /// @dev Locks the receive function. This can be used to prevent stuck ether
     ///      from ending up in the contract but still allowing refunds to be
     ///      received.
-    bool private isReceiveLocked = true;
+    ///      (1) - Unlocked
+    ///      (2) - Locked
+    uint256 private isReceiveLocked = 2;
```

Comment:
I like how OpenZeppelin used constants for these values instead of magic numbers. Can we do that instead?